client/src/components/charts/GekichuScoreChart.tsx

```diff
@@ -22,21 +22,39 @@ const formatTime = (s: DatumValue) =>
 		.toString()
 		.padStart(2, "0")}`;
 
-const scoreToLamp = (s: number) => {
-	switch (s) {
-		case 970000:
-			return "S";
-		case 990000:
-			return "SS";
-		case 1000000:
-			return "SSS";
-		case 1007500:
-			return "SSS+";
+const scoreToLamp = (game: Game) => (s: number) => {
```

Comment:
This is a scarily misleading function name given what the function does. `getScoreYAxisNotch` or something like that would be better.